enhancements/authentication/pod-security-admission.md

```diff
@@ -0,0 +1,113 @@
+---
+title: pod-security-admission
+authors:
+  - "@s-urbaniak"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-09-14
+last-updated: 2021-09-14
+status: informational
+see-also:
+replaces:
+superseded-by:
+---
+
+# PodSecurity admission in OpenShift
+
+Kubernetes recently gained PodSecurity policy admission as part of KEP 2579 [1].
+It is a new pod security admission plugin to enforce Kubernetes PodSecurity standards [2].
+It is meant to replace the existing PodSecurityPolicy admission mechanism in Kubernetes.
+
+This enhancement describes the migration path to integrate PodSecurity admission inside OpenShift.
+Special interest is put to design a migration scheme which lets PodSecurity coexist with the existing Security Context Constraints (SCC) mechanism.
+
+[1] https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/2579-psp-replacement
+[2] https://kubernetes.io/docs/concepts/security/pod-security-standards/
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Motivation
+
+In contrast to PodSecurityPolicy admission the desire is not to disable the new upstream PodSecurity admission plugin subsystem within OpenShift.
+The new PodSecurity admissions plugin is a validating plugin only. This implies the opportunity to let it coexist with the existing SCC logic.
+
+### Goals
+
+[1] Enable PodSecurity admission
+[2] Design an architecture that lets PodSecurity coexist with SCC logic
+
+### Non-Goals
+
+[1] Break existing SCC API
```

Comment:
```suggestion
[1] Deprecate existing SCC API
```

?

break is never an option, but it's important to understand if we still think SCC has its own role to play or not.

Reply:
Agreed on the suggestion, this sounds better :+1: SCCs will definitely stay and we must not break that functionality. Do you think it is advisable to add a paragraph that empasizes this?

Reply:
i think your existing sentence: 
```
Special interest is put to design a migration scheme which lets PodSecurity coexist with the existing Security Context Constraints (SCC) mechanism.
```

is sufficiently clear that SCCs aren't going anywhere.